csc-overrides/partials/footer.html

```diff
@@ -72,22 +29,21 @@
           from 8.30 a.m. to 4 p.m.
         </div>
         <div class="csc-footer__body">
-          {% for link in service_links %}
-            <span class="twemoji">
-              {% include link.icon %}
-            </span>
-            <a class="csc-footer__link" href="{{ link.url }}" target="{{ link.target }}">{{ link.text }}</a>
-            <br>
-          {% endfor %}
+          <ul>
+            <li>{{ footer_link("Contact form", "https://research.csc.fi/support", icon="form-textbox") }}</li>
+            <li>{{ footer_link("servicedesk@csc.fi", "mailto:servicedesk@csc.fi", icon="email") }}</li>
+            <li>{{ footer_link("+358 9 4572821", "tel:+35894572821", icon="phone") }}</li>
+          </ul>
         </div>
       </div>
       <div class="csc-footer__column">
         <div class="csc-footer__heading">Info</div>
         <div class="csc-footer__body">
-          {% for link in info_links %}
-            <a class="csc-footer__link" href="{{ link.url }}" target="{{ link.target }}">{{ link.text }}</a>
-            <br>
-          {% endfor %}
+          <ul>
+            <li>{{ footer_link("Maintenance breaks", "https://research.csc.fi/service-breaks/" ) }}</li>
+            <li>{{ footer_link("Accessibility statement", page_url("support/accessibility.md"), target="_self") }}</li>
+            <li>{{ footer_link("Customer communications archive", page_url("support/contact.md")+"#archives", target="_self") }}</li>
```

Comment:
Or should it be "User communications archive" (end-users vs customers == ministry and HEIs)? Or simply "News(letter) archives"? It was noted that "Mailing list archives" might something users don't click, but having the word "News" in a way or another in the link text might perhaps spark more interest?

Reply:
I changed it to _User communications archives_.

I can't seem to come up with a way to incorporate the word "news" into it. Feel free to WIP this if you feel we should try harder. :slightly_smiling_face: